widgets/data_editor.py

```diff
@@ -711,12 +711,17 @@ def setup_widgets(self):
         self.end = self.add_multiple_decimal_input("End", "end", ["x", "y", "z"],
                                                      -inf, +inf)
 
-        self.unk1 = self.add_integer_input("Unknown", "unk1",
+        self.unk1 = self.add_integer_input("Shortcut Point ID", "unk1",
                                            MIN_UNSIGNED_BYTE, MAX_UNSIGNED_BYTE)
         self.unk2 = self.add_checkbox("Unknown Flag", "unk2",
                                       0, 1)
-        self.unk3 = self.add_checkbox("Unknown Flag 2", "unk3",
+        self.unk3 = self.add_checkbox("Double-sided", "unk3",
                                            0, 1)
+        self.unk4 = self.add_checkbox("Lap Checkpoint", "unk4",
+                                           0, 1)
```

Comment:
The following line needs to be added so that toggling the **Lap Checkpoint** checkbox triggers a viewport update, ensuring that the correct color is used for the checkpoint:

```python
self.unk4.toggled.connect(self.catch_text_update)
```